### DISCUSSION TITLE
What is the most recent release for self-hosters?

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#6122] [Help] [answered]
I see many new features are released, but my version is still 1.0.105-alpha.8 and it says it's up to date:

![image](https://user-images.githubusercontent.com/93232582/171074198-49377fd4-27c8-4b08-873b-e6ca68857fd0.png)

Is it only me or something's wrong with my installation?

**Comment by shogunpurple** [comment, marked as answer]
@dannyshin76 - that looks like a development version. Did you update your `docker-compose.yaml` manually to pull in the dev version?

What do the `image: ` keys look like in your `docker-compose.yaml`? You may need to reset them to the original:
`budibase/apps`
`budibase/worker`

Thanks!
